src/protocol_misc.c

```diff
@@ -110,7 +110,7 @@ bool pong_h(meshlink_handle_t *mesh, connection_t *c, const char *request) {
 	/* Successful connection, reset timeout if this is an outgoing connection. */
 
 	if(c->outgoing) {
-		reset_outgoing(outgoing);
+		reset_outgoing(c->outgoing);
```

Comment:
I don't know how this happened... it was correct on the master branch, and I just cherry-picked it onto the develop branch, but I guess I forgot to run make to check that it actually compiled.

Reply:
Actually the mistake was mine, I couldn't apply the patch so I cherry-picked your changes and missed it. So added it in other commit.